todo_app/data/session_items.py

```diff
@@ -1,19 +1,32 @@
 from flask import session
-
+import todo_app.data.trelloRequests
 _DEFAULT_ITEMS = [
     { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
     { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
 ]
 
 
+_TRELLO_ITEMS = []
+
+
+def update_items():
+    boardID = "645106104aeafb96fe026623"
+    cards = todo_app.data.trelloRequests.get_cards(boardID)
+    for i in range(0,len(c/b/snhxHvkv/devopsboardards)):
+        _TRELLO_ITEMS.append({"id":i, "trelloId":cards[i]["id"], "status":"Not Started", "title": cards[i]["name"]})
+
 def get_items():
     """
     Fetches all saved items from the session.
 
     Returns:
         list: The list of saved items.
     """
-    return session.get('items', _DEFAULT_ITEMS.copy())
+    #return session.get('items', _DEFAULT_ITEMS.copy())
+    #return session.get('items', _TRELLO_ITEMS.copy())
```

Comment:
I'd recommend getting in the habit of deleting commented out code as soon as it's no longer immediately useful - we can always recover it with Git later if we need to, but the cruft makes the code harder to read right now